Assignment 2.Rmd

```diff
@@ -2,8 +2,13 @@
 
 ## Part I
 Start by installing the "igraph" package. Once you have installed igraph, load the package.
-
+```{r}
+install.packages("igraph")
```

Comment:
No need to install the package every time, but you need to "library" it every time you reopen R.